File: Work/Cinda/cinda-notes/assets/日記_2020_0715/file/b2e/model/AccountFunctionDisabled.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Mxic.ITC.PAM.Model.Entity
{
    public class AccountFunctionDisabled
    {   
        public decimal Id { get; set; }
        public decimal? SignFormId { get; set; }
        public string FormStatus { get; set; }
        public string FormType { get; set; }
        public DateTime? CloseDate { get; set; }
        public string Department { get; set; }
        public string EmpNo { get; set; }
        public string Name { get; set; }


    }
}

File: Work/Cinda/cinda-notes/assets/日記_2020_0715/file/f2e/model/AccountFunctionDisabled.ts
export default class AccountFunctionDisabled {
    Department: string;
    UserEmpNo: string;
    User: string;
    CloseDate: Date;
    Status: number; // FormStatus: 文件處理狀態
    // 0 已派工
    // 1 待主管送件
    // 2 中止
    // 3 結案
    F0RM_TYPE: number; // 離職帳號或權限停用單狀態
    Item: string; // FormType
    // 0 沒有選擇
    // 1 權限提前停用，但AD/Note/Novell帳號及設備不提前及繳回
    // 2 AD / Note / Novell帳號提前停用及設備提前繳回
    // 3 帳號與權限皆不提前停用
    SignFormId: number;
}
